src/mono/wasm/runtime/crypto-worker.ts

```diff
@@ -358,7 +355,17 @@ class LibraryChannel {
         return String.fromCharCode.apply(null, slicedMessage);
     }
 
-    private acquire_lock() {
+    // Make sure to never call any blocking code in the callback
+    private _using_lock_for_nonblocking_cb(cb: Function) {
+        try {
+            this._acquire_lock_with_spin_wait();
+            return cb();
+        } finally {
+            this._release_lock();
+        }
+    }
+
+    private _acquire_lock_with_spin_wait() {
```

Comment:
Is it common to put implementation details in the name of the method? "With spin wait" seems more appropriate for a comment than apart of the method name.

Similar comment for `_using_lock_for_nonblocking_cb`.